sdk/tests/conformance2/rendering/blitframebuffer-filter-outofbounds.html

```diff
@@ -111,6 +115,10 @@
             [4, 4, 8, 8],
             [8, 8, 8, 8],
             [6, 6, 8, 8],
+            [5, 5, 8, 8],
```

Comment:
I believe this should start at 4 instead of 5, as (4, 4) will sample from (0.25, 0.25) which is valid.